### PR TITLE
pull request: added definition for pathjs

### PR DESCRIPTION
**Pull request by lokeshpeta**

### CODE REVIEW

File: pathjs/path.d.ts [hunk 1]
```diff
@@ -0,0 +1,53 @@
+// Type definitions for Pathjs v0.8.4
+// Project: https://github.com/mtrpcic/pathjs/blob/master/path.js
```
Comment (horiuchi) [inline pathjs/path.d.ts]: I think the project url of `https://github.com/mtrpcic/pathjs` is good.

**Comment by horiuchi** [comment]
@lokeshpeta Thank you for your contribution.
Please rename to `pathjs/pathjs.d.ts`.

**Comment by lokeshpeta** [comment]
@horiuchi Thanks you very much for comments 
done the renaming

**Comment by horiuchi** [comment]
@lokeshpeta Thanks mate!

**Pull request merged**